README.md

```diff
@@ -107,6 +106,103 @@ print(myShoppingList)
+
+Whilst arrays share similar concepts and principles across programming languages, you create and use them differently in different languages.
+
+## Creating an Array
+
+Here's how to create an array of values in Swift:
+
+```swift
+var list = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+In this case, you are creating a variable with the `var` keyword called 'list'. Swift will decide on the type of values allowed in the array for you as you add them.
+
+To be more precise, you can declare the type of values allowed in the array explicitly:
+
+```swift
+var shoppingList: [String] = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+Here you are creating a variable with the `var` keyword called 'shoppingList', setting the type of values allowed explicitly to be of `String` and then adding five string values to the array.
+
+Or alternatively (but less used):
+
+```swift
+var shoppingList: Array<String> = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+Often an array is initialized first and populated later:
+
+```swift
+var shoppingList: [String]
+shoppingList = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+Here you create an array variable called 'shoppingList' that will accept values of type `String`. You then assign five string values to the variable.
+
+## Constant Arrays
+
+As you have used the `var` keyword in all the examples so far, these collections of values are _mutable_. This means you can add, remove or change items in the array. As with other variable types in Swift, to make an array constant, unchangeable or _immutable_, use the `let` keyword.
+
+For example, a shopping list may change, but the ingredients for a cheese sandwich never will:
+
+```swift
+let cheeseSandwich: [String] = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+## Other Values in an Array
+
+As you would expect, arrays can contain values apart from strings, and created the same way. For example, to create an array recording attendance at your Saturday afternoon sandwich meet:
+
+```swift
+var afternoonAttendance: [Int] = [10, 5, 15, 2, 12]
+```
+
+## Accessing Items in an Array
+
+You have an array of values, to access an individual item, you need to know it's location in the collection. For example, in the `shoppingList` array, you want to print the value of the item in the second position, starting from 0, this is position 1\. Below, you assign the value in that position to a variable, and print the value of the variable.
+
+```swift
+let itemAtIndexOne = shoppingList[1]
+
+print(itemAtIndexOne)
+```
+
+## Changing Items in an Array
+
+You have decided instead to make a ham sandwich, so you need to change the 'Cheese' value in the array. Again you need to know the position of 'Cheese' and change it by assigning a new value:
+
+```swift
+shoppingList[2] = "Ham"
+```
+
+If you now print the array, the values will have changed:
+
+```swift
+print(shoppingList)
+```
```

Comment:
Again, similar to my above comment. Put a comment below this print function to show the student what actually is being printed to the console.